TOOODO/Data/Sources/DataSource/Local/CategoryLocalDataSource.swift

```diff
@@ -8,15 +8,17 @@
 import Foundation
 import RealmSwift
 
+import Combine
+
 public protocol CategoryLocalDataSource {
-    func getCategories() -> Array<CategoryDto>
+    func getCategories() -> AnyPublisher<Array<CategoryDto>, Error>
```

Comment:
combine 말고 추후에 Async/Await로 리펙토링 해보시는 건 어떤가요?? 저는 이게 더 직관적이여서 좋아합니다
```suggestion
    func getCategories() async throws -> CategoryDto
```

Reply:
다 바꾸셔야 돼서 번거롭기 때문에 시간이 된다면 추천입니다~

Reply:
처음에 async await을 사용했다가 데이터가 변경될 때마다 갱신하기 위해 AnyPublisher로 변경했습니다.